src/form.js

```diff
@@ -95,9 +95,12 @@ export default function form({fields, validate = () => ({})}) {
         forceValidate: () => this.touch(fields),
         values: () => this.state.values,
         onValues: values => {
-          if (!this.props.value) {
-            this.setState({ values });
+          this.setValues(values);
```

Comment:
I am not sure we want to set these values if there is a props.value being passed here.  If there is, it will overwrite whats there.  It might be better to retain the `!this.props.value` check.  In this case, the event should be fired to the parent thats controlling the `value` prop and allow them to pass the proper values down.  (When new values are passed, `setValues` is called, so if the parent is just passing the same values after an `onChange` this will have the same effect).